eth_tester/utils/transactions.py

```diff
@@ -8,7 +8,7 @@
     'from',
     'to',
     'gas',
-    'gasPrice',
+    'gas_price',
     'value',
     'data',
     'nonce',
```

Comment:
Will that make sense to remove `chainId` in the next line? 
It seems not the param of transaction, at least for now.

Reply:
Thanks @mhchia 

I *suspect* this was trying to address the same issue as https://github.com/ethereum/web3.py/pull/656 which is now merged, so I think you're right that this can be removed.

Reply:
Okay I see. I didn't know that issue.
Thank you for pointing it out